src/tic_tac_toe/session.clj

```diff
@@ -0,0 +1,47 @@
+(ns tic-tac-toe.session
+  (:require [tic-tac-toe.display :refer :all]
+            [tic-tac-toe.board :refer :all]
+            [tic-tac-toe.player :refer :all]
+            [tic-tac-toe.rules :refer :all]
+            [tic-tac-toe.messages :refer :all]))
```

Comment:
You should avoid doing :refer :all unless there's maybe just one namespace that you're importing & it's very closely related to the namespace you're using it in. I prefer to do something like `rules/winner?` instead

Reply:
So I changed [tic-tac-toe.rules :refer :all] to [tic-tac-toe.rules :as rules]

Then in my session file, I replaced winner? with rules/winner?
I'll do the same with the others as well.